path/src/keybindings.ps1

```
@@ -0,0 +1,251 @@
+	if (-not $IsWindows) { return }
+	$manifestPath = get_terminal_keybindings_manifest_path
+	$windowsTerminalSettingsPaths = [System.Collections.Generic.List[string]]::new()
+	$editorKeybindingsPaths = [System.Collections.Generic.List[string]]::new()
+
+	if (Test-Path $manifestPath) {
+		try {
+			$manifest = Get-Content $manifestPath -Raw -Encoding UTF8 | ConvertFrom-Json
+			if ($manifest.windowsTerminalSettings) { $manifest.windowsTerminalSettings | ForEach-Object { $windowsTerminalSettingsPaths.Add($_) } }
+			if ($manifest.editorKeybindings) { $manifest.editorKeybindings | ForEach-Object { $editorKeybindingsPaths.Add($_) } }
+		}
+		catch { <# ignore #> }
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)